.github/workflows/run_tests.yaml

```diff
@@ -0,0 +1,17 @@
+name: run-tests
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
```

Comment:
I'm not too familiar with Firebase, but it looks like we would run the app on linux.

Reply:
Same strategy for the backend works here.